README.md

```diff
@@ -110,6 +110,26 @@ You will need to add `$HOME/.local/bin` in your `$PATH` to be able to run umu th
 export PATH="$HOME/.local/bin:$PATH"
 ```
 
+# Packages:  
+## NisOS
+if you want to add umu-launcher as a flake add this to your inputs in flake.nix
+```nix
+  inputs = {
+    umu= {
+      url = "git+https://github.com/Open-Wine-Components/umu-launcher/?dir=packaging\/nix&submodules=1";
+      inputs.nixpkgs.follows = "nixpkgs";
+    };
+  }
+```
+and in your configuration.nix (or obv any file in your config)
```

Comment:
Obv -> Obviously

It is a Readme, needs some formality or at least not abbreviated words. Someone might try translate it through an automated tool. IMHO this needs to be rephrased.

Reply:
you do not need to have the first letter as capital. Auto-correct did it for me. lemme fix that

Reply:
I fixed this I will keep this thread open though
for well (I'll probably think of a suggestion)
and sorry for not responding right away I was messing around with git branches
and I messed up lol https://github.com/Open-Wine-Components/umu-launcher/pull/77
anyway I'll think of better wording
and FWIW in nix you can really put that wherever you want

Reply:
>  and FWIW in nix you can really put that wherever you want

From experience, users have a tendency to misinterpret instructions in very creative ways. I think that just mentioning the proper/default/documented way of doing it is enough. Users that know where they can deviate from instructions with alternative ways to do something, will use them despite the documentation. For the rest of them, using the "proper" way helps with consistency and reproducibility.

Reply:
@loathingKernel than I guess I will just remove that obvious section now
if you think this is enough I guess resolve this thread
and this is ready to be merged probably

Reply:
What it should say is up to you, you are the knowledgeable person with how NixOS works. I am merely giving suggestions on how instructions should be structured amd formatted. You have to figure out what is relevant and what isn't in this case because it is unlikely these instructions can be later edited by us while retaining your intended information. For me, once you are happy with the instructions, this conversation can be marked as resolved.

Reply:
FWIW I used very generic names most people in the nix sphere will know what they are doing so yeah the state the nixos section is in now is good enough to be honest

and also to be honest I'm sure this will not be the final stage of that section so yeah this can be resolved